tensorflow/lite/micro/kernels/svdf.cc

```diff
@@ -22,6 +22,7 @@ limitations under the License.
 #include "tensorflow/lite/kernels/internal/common.h"
 #include "tensorflow/lite/kernels/internal/quantization_util.h"
 #include "tensorflow/lite/kernels/internal/tensor_ctypes.h"
+#include "tensorflow/lite/kernels/internal/tensor_utils.h"
```

Comment:
Looks to me like this is bringing in some non-TFLM compatible dependencies. Perhaps try using ```tensorflow/lite/kernels/internal/reference/portable_tensor_utils.h```

Reply:
Ah thanks, yes will try that :+1:

Reply:
I've updated the PR in 9c8f7f7. Is there a way that I can run the TFLM tests locally to test small changes like this?

Reply:
It should be possible to run ```make -f tensorflow/lite/micro/tools/make/Makefile test``` to catch some issues like this.

Sometimes issues pop up on certain targets and not others. In this case, I tend to look at the test log to see the last make or bazel command which caused the error. Running that command locally should reproduce the same error most of the time.